### DISCUSSION TITLE
Add prominent "Sign out" button (or link) to the Dashboard

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#581] [Feature Requests]
To sign out (A.K.A. logout) the customer needs to click their profile in the upper right hand corner and select "Sign out" from the UI which appears. Given the wealth of sensitive information contained in the SimpleLogin tool I believe this function should be promoted to a higher level. Specifically I believe that logout should be directly visible as a button or link on every page and should not be behind another UI element.

![SL Sign Out Link](https://user-images.githubusercontent.com/88680678/129457676-4d326c46-dcbe-4e32-857f-99cb6d14a0f6.jpg)

**Comment by nguyenkims** [comment]
@WhamboMPS From what we see on other apps (ProtonMail, Gmail, etc) the logout button usually appears when clicking on the profile picture. This is to avoid taking up space in the UI for an action that isn't frequently used. In the future, we can add an option for the session duration so users can choose to automatically log out after several hours or immediately when closing the browser tab.
